### PR TITLE
Add a new wraith ability: woosh swoosh

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Add a new ability for wraith, rewind. 

Click on a xeno/human, this saves his position and stats. If he is not dead after 5 seconds, the ability will transport it back to where it was, with the health it had.

## Why It's Good For The Game

Woosh swoosh

## Changelog
:cl:
add: New wraith ability, rewind
/:cl:

### CODE REVIEW

**Comment by SilencedMP5A5** [comment]
_**KILLER QUEEN! BITES ZE DUSTO!**_

**Comment by Losenis** [comment]
Woosh Swooshing marines back into the tadpole landing area they thought they were clear of

**Comment by Kuro020** [comment]
this ability name is dumb

**Comment by Lauwma** [comment]
So if they took damage do they heal ? Same thing for fractures ?

**Comment by private-tristan** [comment]
welcome to a true bean's world

File: code/modules/mob/living/carbon/xenomorph/castes/wraith/abilities_wraith.dm [hunk 1]
```diff
@@ -567,3 +567,81 @@ GLOBAL_LIST_INIT(wraith_banish_very_short_duration_list, typecacheof(list(
...
+	UnregisterSignal(targeted, COMSIG_MOVABLE_MOVED)
+	if(QDELETED(targeted) || targeted.stat == DEAD)
+		targeted = null
+		return
+	targeted.status_flags |= INCORPOREAL|GODMODE
+	INVOKE_NEXT_TICK(src, .proc/rewind)
+	targeted.canmove = FALSE
+	playsound(targeted, 'sound/effects/woosh_swoosh.ogg')
+
+/datum/action/xeno_action/activable/rewind/proc/rewind()
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/castes/wraith/abilities_wraith.dm:628]: Autodoc

**Comment by TiviPlus** [comment]
Also no modified icons?

**Comment by TiviPlus** [comment]
Autodoc review still pending but I'll give it tmc

**Comment by InterroLouis** [comment]
Ahem. No. That's all. Just no.

**Comment by Mechanelize** [comment]
image
use this, kuro approved

**Review by Mechanelize — CHANGES_REQUESTED** [review]
use modern ability sprite instead of old cm one that doesn't match art syle, provided on discord

**Comment by Arctrooper8** [comment]
Can you make it so that wraith isn't able to kidnap people across Z levels? For example: if you're sitting and tadpole and it launched you can get pulled back all the way to groundside when Tad is in the air.

**Comment by Dolth** [comment]
Looks completely broken on paper and unfun for marines but extremly fun for benos so, ehhh I am gonna play wraith and try this for the upcoming rounds.

**Comment by vilereaver41** [comment]
Why are you turning wraith into a support caste? that is boring. the intention of wraith was to be an infiltrator and you pretty much killed it. the playstyle from the way i see you made wraith is you assume the wraith is going to find a xeno and stick with it to support them. that barely if ever happens

**Comment by Mechanelize** [comment]
>the intention of wraith was to be an infiltrator and you pretty much killed it.

It's a new ability with no other changes, it's a straight buff and doesn't hurt its ability as an infiltrator, just gives it the option to play a more support role, same with the portal ability.

**Comment by Losenis** [comment]
I haven't realized how busted this is with primo bull until I faced it. Primo bull will run through the entire server hitting everyone as fast as it can run into them unlike slashing which has a cooldown, runs at 10+ tiles per second in all directions making hitting it extremely difficult, and with the ability now you only have like 3 or 4 seconds to kill it or it escapes. Not to mention wraith isn't a max 1 alive xeno like queen or shrike so this ability can be chained if there's 2 wraiths or more.

**Review by Kuro020 — APPROVED** [review]
disappointed that its not actually named woosh swoosh

**Comment by zzurge** [comment]
I miss original wraith

**Pull request merged**